apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -596,6 +620,16 @@
                         "max_time" : {
                             "type" : "integer",
                             "default" : 0
+                        },
+                        "rate_modifiers" : {
+                            "type" : "string",
+                            "default" : "mempool.size: 10",
```

Comment:
over engineering, but fine.

Reply:
I guess we'll see... ;-)
At least when working under heavy load with mnesia, detecting overload and applying backpressure can be a vital part of load tolerance.